# goal
geometry_msgs/Pose tool_pose
---
# result
bool result
---
# feedback
sensor_msgs/JointState joint_step